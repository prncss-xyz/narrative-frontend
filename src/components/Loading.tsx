import { H3 } from "./basics";

/*
 * what appears when page datas are loading
 * could be a fancy spinner
 */
export function Loading() {
  return <H3>Loading...</H3>;
}
