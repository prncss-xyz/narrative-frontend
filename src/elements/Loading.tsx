import { H2 } from "./basics";

export function Loading() {
  return <H2>Loading...</H2>;
}
