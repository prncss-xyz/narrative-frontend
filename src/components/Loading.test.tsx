import { render } from "@testing-library/react";
import { Loading } from "./Loading";

describe("Loading", () => {
  it("should render correctly", () => {
    const { container } = render(<Loading />);
    expect(container).toMatchSnapshot();
  });
});
