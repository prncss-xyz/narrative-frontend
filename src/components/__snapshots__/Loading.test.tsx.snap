// Vitest Snapshot v1

exports[`Loading > should render correctly 1`] = `
<div>
  <h3
    class="css-5jl3ge-Box e2i8k6n3"
    color="muted"
    font-size="1"
    font-weight="normal"
    text-decoration="underline"
  >
    Loading...
  </h3>
</div>
`;
